### PR TITLE
Kong Mesh installation docs

### PR DESCRIPTION
**Pull request by lena-larionova**
Issue tracked in https://konghq.atlassian.net/browse/DOCS-1304.

Previous state of the install docs: we gave users some vague instructions, expecting them to cross-reference the Kuma docs. For a better (less confusing) user experience, this PR brings all the installation instructions over from the Kuma doc site and adjusts them for Kong Mesh. 

For reference, see each topic in Kuma docs, eg https://kuma.io/docs/0.7.2/installation/kubernetes/.

**Preview:** all topics under https://deploy-preview-2368--kongdocs.netlify.app/mesh/1.0.x/install/ except Helm.

### Universal install instructions
Ubuntu, CentOS, MacOS, Debian, Amazon Linux, and Redhat installation instructions are exactly the same after downloading the binaries. Therefore, they're documented through `include` sections. 

### Kubernetes, OpenShift, and Docker
These installation topics have some differences between each other, enough that using shared instructions doesn't work. 

### Comments/Notes
Writing these topics brought up some formatting issues in the docs. In subsequent PRs, I'll be adjusting the styling for codeblocks within notes:

Screen Shot 2020-10-09 at 11 30 57 AM

and for tabs:
Screen Shot 2020-10-09 at 11 31 18 AM

Final styling tbd.

### CODE REVIEW

**Review by tlblessing — APPROVED** [review]
A few comments for thought.

File: app/mesh/1.0.x/installation/centos.md [hunk 1]
```diff
@@ -37,24 +41,8 @@ $ tar xvzf kong-mesh-{{page.kong_latest.version}}*.tar.gz
...
-This can be done by running the control plane with:
-
-```sh
-$ KUMA_LICENSE_PATH=license.json kuma-cp run
-```
-
-Where `license.json` is the path to a valid {{site.mesh_product_name}} license file on the file system.
-
-### 3. Follow Kuma Instructions
+{% include /md/mesh/1.0.x/install-universal-run.md %}
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/mesh/1.0.x/installation/centos.md:44]: If you're referring to the "universal" label, that's the type of Mesh install. The Kuma docs are divided into Universal and Kubernetes guides.

**Review by nickolaev — APPROVED** [review]
Approving with 2 remarks:
 * @subnetmarco needs to approve this one too
 * What will be the procedure to update these docs in the future? Say we have a significant change in the Mesh resource (which is referenced here), how this gets propagated to the docs?

**Comment by lena-larionova** [comment]
> * What will be the procedure to update these docs in the future? Say we have a significant change in the Mesh resource (which is referenced here), how this gets propagated to the docs?

You can either let me know and I'll work with whoever I need to to update the docs, or you can edit the docs directly. If we have a changelog, that will help the docs team immensely, as we'll at least know when things are changing. 

Every doc page has a link to github in the top right, you're welcome to create PRs that way as needed:

Screen Shot 2020-10-15 at 4 41 38 PM

**Review by subnetmarco — APPROVED** [review]

**Pull request merged**